npm2deb/__init__.py

```diff
@@ -224,6 +226,10 @@ def create_upstream_metadata(self):
             content = utils.get_upstream_metadata('github') % args
             utils.create_dir("debian/upstream")
             utils.create_debian_file('upstream/metadata', content)
+        elif self.upstream_repo_url.find('gitlab') >= 0:
```

Comment:
Do we need to add metadata for `fakeupstream/npmregistry` too ?

Reply:
This needs to parse package.json to find github url